livestyled/models/fulfilment_point.py

```diff
@@ -100,6 +101,15 @@ def __init__(
         self.reference = reference
         self.external_id = external_id
 
+        audience_iris = []
+        for audience in audiences:
```

Comment:
I don't think this is required, its a half attempt at field validation

I could still call this function with [12, 'user_management/audiences/2', 'somerandomiri/13'] and that would be valid. Lets just make the function require an array of iri's and not do any specific formatting.